lldb/test/API/lang/swift/unit-tests/xctest.c

```diff
@@ -17,7 +17,10 @@
 
 int main(int argc, const char **argv)
 {
-  char *dylib = strcat(dirname(argv[0]),"/UnitTest.xctest/Contents/MacOS/test");
+  char dylib[4096];
```

Comment:
I'm sure it doesn't really matter, but should these buffers us `PATH_MAX`?

```suggestion
  char dylib[PATH_MAX];
```

Reply:
Thanks!